docker/artemis-migration-check-mysql.yml

```diff
@@ -0,0 +1,39 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Artemis-Migration-Check-MySQL Setup
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    artemis-app:
+        extends:
+            file: ./artemis.yml
+            service: artemis-app
+        ports:
+            - "8080:8080"
```

Comment:
```suggestion
```
Do we need access to the Artemis instance? If not I guess we can delete the ports or limit it to `127.0.0.1`? I guess we just check the exit code anyhow.

Reply:
Good point, I will remove it